File: LeetCodeProblems/1. Two Sum.py
from typing import List

class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        dic = {}
        for i,n in enumerate(nums):
            dif = target - n
            if dif in dic:
                return [dic[dif],i]
            dic[n] = i

File: Leet Go/1. Two Sum.go
package main

func twoSum(nums []int, target int) []int {
	for i, number1 := range nums {
		for j := i + 1; j < len(nums); j++ {
			if nums[j]+number1 == target {
				return []int{i, j}
			}
		}
	}
	return nil
}
